src/web/components/App.tsx

```diff
@@ -166,7 +166,9 @@ export const App = ({ CAPI, NAV }: Props) => {
     // We should monitor this function call to ensure it only happens within an
     // article pages when other pages are supported by DCR.
     useEffect(() => {
-        incrementWeeklyArticleCount();
+        if (getCookie('gu_article_count_opt_out') == null) {
```

Comment:
let's add a const for `"gu_article_count_opt_out"` to here https://github.com/guardian/dotcom-rendering/blob/master/src/web/lib/contributions.tsx#L15

Reply:
Yeah great shout - I've added the const + added a hasOptedOutOfArticleCount function to clean up the logic a little